Add fallbackToSmaxage flag to CustomTtlListener and check for isCacheable before calling store (#575)
FriendsOfSymfony/FOSHttpCache: doc/symfony-cache-configuration.rst

    new CustomTtlListener('My-TTL-Header');

The custom header is removed before sending the response to the client.
You can enable keeping the custom header with the `keepTtlHeader` parameter::

>     new CustomTtlListener('My-TTL-Header', keepTtlHeader: true);

Reviewer: thanks for this one :+1:

> By default if the custom ttl header is not found it will fallback to s-maxage.

Reviewer: ```suggestion
By default if the custom ttl header is not present, the listener falls back to the s-maxage cache-control directive.
```

> To disable this behavior you can set the `fallbackToSmaxage` parameter to false::

Reviewer: ```suggestion
To disable this behavior, you can set the `fallbackToSmaxage` parameter to false::
```

>     new CustomTtlListener('My-TTL-Header', keepTtlHeader: true, fallbackToSmaxage: false);

Reviewer: when using named parameter, we don't need the one we are not concerned with, i think?
```suggestion
    new CustomTtlListener('My-TTL-Header', fallbackToSmaxage: false);
```
